backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -462,6 +462,16 @@ public void deleteOrganization(Organization org) {
     return getOrganizationRoleClaimsForUser(user);
   }
 
+  // returns a set of the email addresses of all members of a group
+  public boolean isEmailInGroup(final String groupId, final String email) {
+    final Group oktaGroup = _client.getGroup(groupId);
```

Comment:
This may be a question to resolve in #1214, but can we make this determination without making a call to Okta? Looking at the token at I get when logged into `dev.simplereport.gov`, I see that there is an `{environment}_roles` payload claim in the token that should tell us without an additional service call whether the bearer is in the admin group for a given environment:

```
  "dev_roles": [
    "SR-DEV-TENANT:example.com:ALL_FACILITIES",
    "SR-DEV-TENANT:example.com:USER",
    "SR-DEV-TENANT:example.com:NO_ACCESS",
    "SR-DEV-ADMINS"
  ],
```

Reply:
Yes, we should be able to get this information out of the existing authentication token.

Reply:
Good point, this was causing a lot of unnecessary querying of Okta.  I added #1214 to this PR to address it.